src/app/forms/templates/psv/psv-body.template.ts

```diff
@@ -1,7 +1,7 @@
 import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeEditTypes, FormNodeTypes } from '../../services/dynamic-form.types';
 
-export const psvBodyTemplate: FormNode = {
+export const PsvBodyTemplate: FormNode = {
```

Comment:
Unsure why this const was capitalised

Reply:
Because They're all capitalized except for psv, tyres and hgv and it was annoying me, but I didn't want to amend tyres and hgv because it felt like a dick move, so now theyre the only ones that aren't

Reply:
Yeah leave this, I think if I get some free time I am going to go through and re name all the templates + refactor any that aren't just JSON, will cover this then.